python/sdk/merlin/fluent.py

```diff
@@ -322,6 +322,21 @@ def log_model(model_dir):
     _check_active_model_version()
     _active_model_version.log_model(model_dir)  # type: ignore
 
+def log_custom_model(model_dir: str, image :str, command: str, args: str):
```

Comment:
The signature of this function must be consistent with `ModelVersion#log_custom_model`